Styles.css

```diff
@@ -0,0 +1,287 @@
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Othergit tsa
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+:root {
+    /*engels vertalen*/
+    --Purple: #5722CD;
+    --Blue: #4845E4;
+    --Pink: #ED4385;
+    --Black: #0C0C0C;
+    --Lightpurple: #F2F0FF;
+    --Lightgrey: #F3F6F9;
+    --Grey: #7A7A7A
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+h1, h5 {
+    font-family: 'Merriweather', serif;
+    font-size: 40px;
+}
+
+h2, h3, h4 {
+    font-family: 'Merriweather', serif;
+    font-size: 30px;
+}
+
+
+button {
+    background-color: var(--Pink);
+    font-family: 'roboto', sans-serif;
+    border-radius: 40px;
+    height: 60px;
+    width: 200px;
+    box-shadow: 0 8px 24px rgba(0, 0, 0, 0.3);
+    color: var(--Lightgrey);
+    border: none;
+    padding: 12px 24px;
+    text-transform: uppercase;
+
+}
+a:link {
+    text-decoration: none;
+}
+
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+
+
+
+
+/* --------------- 3.1 Reusables ----------------------- */
+.outer-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    /*border: 5px solid var(--Purple);*/
+}
+
+.inner-container {
+    flex-basis: 300px;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    text-align: center;
+    margin: 40px 350px;
+    border: 5px solid orange;
+}
+
+
+
+.text-settings {
+
+}
+
+.article-settings-section-2 {
+
+}
+
+/* --------------- 3.2 Other --------------------------- */
+
+
+
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+
+/* --------------- 4.1 Header ------------------------- */
+.inner-container-header {
+
+}
+
+
+.header-text {
+    background-color: #6149e8;
+    color: white;
+}
+
+.header-text p {
+    font-size: 24px;
+    font-weight: 300;
+
+}
+
+.header-nav a:link {
+    color: var(--Lightgrey);
+}
+
+.header-nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 100%;
+    padding: 20px 140px;
+}
+
+.header-nav ul {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 400px;
+
+}
+
+.header-nav ul li {
+    font-size: 20px;
+}
+
+.image-header {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+#image\ header {
+    height: 5em;
+}
+
+/* --------------- 4.2 section 1 ------------------- */
+.container-section-1 {
+    color: #4845E4;
+    font-size: 30px;
+    font-family: 'Merriweather', serif;
```

Comment:
Als de tekst in een titel-font moet worden weergegeven kun je er misschien beter h-elementen van maken ;)